hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutManagement.java

```diff
@@ -251,6 +252,11 @@ private Rollout createRolloutGroups(final int amountOfGroups, final RolloutGroup
         return rolloutRepository.save(savedRollout);
     }
 
+    private boolean isUserConsentEnabled() {
```

Comment:
Wondering about the name (which reflects one use case). would have expected sth like "isConfirmationSupportEnabled" or "isAssignmentConfirmationsEnabled"...

Reply:
Will be discussed after implementing all features of the user consent story